android/app/build.gradle

```diff
@@ -41,6 +45,40 @@ android {
         sourceCompatibility JavaVersion.VERSION_1_8
         targetCompatibility JavaVersion.VERSION_1_8
     }
+
+    if (dropboxAppID) {
+        def dropboxActivity = """<activity
+                android:name="com.dropbox.core.android.AuthActivity"
+                android:configChanges="orientation|keyboard"
+                android:launchMode="singleTask">
+            <intent-filter>
+    
+            <!-- Change this to be db- followed by your app key -->
+            <data android:scheme="db-${dropboxAppID}" />
+    
+                    <action android:name="android.intent.action.VIEW" />
```

Comment:
@hristoterezov, the line above looks incorrectly offset to me on GitHub.

Also, both of app's and sdk's AndroidManifest.xml appear to indent by 2 spaces with double indentation for attributes. Could you please comply